<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recipes</class>
 <widget class="QDialog" name="Recipes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recipe Database</string>
  </property>
  <widget class="QLabel" name="label_sec_status">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>410</y>
     <width>130</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>+Status</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_Recipes">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>641</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>55</width>
     <height>15</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>55</width>
     <height>15</height>
    </size>
   </property>
   <property name="text">
    <string>Recipes:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>212</width>
     <height>409</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="label_RecipeID">
      <property name="text">
       <string>Recipe ID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_RecipeID">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_RecipeName">
      <property name="text">
       <string>Recipe Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_Name"/>
    </item>
    <item>
     <widget class="QLabel" name="label_Ingredients">
      <property name="text">
       <string>Ingredients:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_Ingredients"/>
    </item>
    <item>
     <widget class="QLabel" name="label_Instructions">
      <property name="text">
       <string>Instructions:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_Instructions"/>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_Clear">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Delete">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Save">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>400</y>
     <width>272</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit_Search"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Search">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>tableView_Recipes</zorder>
  <zorder>label_6</zorder>
  <zorder>layoutWidget1</zorder>
  <zorder>layoutWidget2</zorder>
  <zorder>label_sec_status</zorder>
 </widget>
 <tabstops>
  <tabstop>lineEdit_Name</tabstop>
  <tabstop>lineEdit_Ingredients</tabstop>
  <tabstop>lineEdit_Instructions</tabstop>
  <tabstop>pushButton_Clear</tabstop>
  <tabstop>pushButton_Delete</tabstop>
  <tabstop>pushButton_Save</tabstop>
  <tabstop>lineEdit_Search</tabstop>
  <tabstop>pushButton_Search</tabstop>
  <tabstop>lineEdit_RecipeID</tabstop>
  <tabstop>tableView_Recipes</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
